src/mixer_tests.cc

```diff
@@ -287,27 +322,51 @@ TEST_F(MixerTest, Throughput) {
 //  material inventory.
 TEST(MixerTests, MultipleFissStreams) {
   std::string config =
-      "<in_commods>"
-      "<val>stream1</val>"
-      "<val>stream2</val>"
-      "<val>stream3</val>"
-      "</in_commods>"
-      "<in_buf_sizes>"
-      "<val>2.5</val>"
-      "<val>3</val>"
-      "<val>5</val>"
-      "</in_buf_sizes>"
-      "<mixing_ratios>"
-      "<val>0.8</val>"
-      "<val>0.15</val>"
-      "<val>0.05</val>"
-      "</mixing_ratios>"
-      "<out_commod>dummyout</out_commod>"
+      "<in_streams>"
+        "<stream>"
+          "<info>"
+            "<mixing_ratio>0.8</mixing_ratio>"
+            "<buf_size>2.5</buf_size>"
+          "</info>"
+          "<commodities>"
+            "<item>"
+              "<commodity>stream1</commodity>"
+              "<pref>1</pref>"
+            "</item>"
+          "</commodities>"
+        "</stream>"
+        "<stream>"
+          "<info>"
+            "<mixing_ratio>0.15</mixing_ratio>"
+            "<buf_size>3</buf_size>"
+          "</info>"
+          "<commodities>"
+            "<item>"
+              "<commodity>stream2</commodity>"
+              "<pref>1</pref>"
+            "</item>"
+          "</commodities>"
+        "</stream>"
+        "<stream>"
+          "<info>"
+            "<mixing_ratio>0.05</mixing_ratio>"
+            "<buf_size>5</buf_size>"
+          "</info>"
+          "<commodities>"
+            "<item>"
+              "<commodity>stream3</commodity>"
+              "<pref>1</pref>"
+            "</item>"
+          "</commodities>"
+        "</stream>"
+      "</in_streams>"
+      "<out_commod>mixedstream</out_commod>"
       "<outputbuf_size>0</outputbuf_size>"
       "<throughput>0</throughput>";
-
+  std::cout << "1" << std::endl;
   int simdur = 1;
```

Comment:
usually we don't write to stdout in tests. Is this an artifact of debugging or should it stay here?